src/snowcli/cli/snowpark_shared.py

```diff
@@ -85,6 +85,7 @@ def snowpark_package(
                     "requirements.other.txt",
                     check_anaconda_for_pypi_deps,
                     package_native_libraries,
+                    install_directory=source / ".packages",
```

Comment:
Should we push it down to implementation instead of doing this logic in argument?

Reply:
Done